source/modules/MEI.stringtab.xml

```diff
@@ -1,51 +1,91 @@
     <attList>
+      <attDef ident="tab.align" usage="opt">
+        <desc xml:lang="en">Attribute that describes the vertical alignment of tablature symbols.
+          Only applicable in cases where the symbols' vertical position does not communicate other
+          information, such as courses (i.e., only in German lute tablature). Typical values are <val>
+            top</val> and <val>bottom</val>.</desc>
+        <datatype>
+          <rng:ref name="data.VERTICALALIGNMENT" />
+        </datatype>
+      </attDef>
+      <!-- Below here is unchecked / legacy -->
       <attDef ident="tab.fing" usage="opt">
-        <desc xml:lang="en">Indicates which finger, if any, should be used to play an individual string. The
-          index, middle, ring, and little fingers are represented by the values 1-4, while <val>t</val> is
+        <desc xml:lang="en">This attribute is deprecated and will be removed in a future version.
+          Indicates which finger, if any, should be used to play an individual string. The index,
+          middle, ring, and little fingers are represented by the values 1-4, while <val>t</val> is
           for the thumb. The values <val>x</val> and <val>o</val> indicate muffled and open strings,
           respectively.</desc>
         <datatype>
-          <rng:ref name="data.FINGER.FRET"/>
+          <rng:ref name="data.FINGER.FRET" />
         </datatype>
       </attDef>
       <attDef ident="tab.fret" usage="opt">
         <desc xml:lang="en">Records the location at which a string should be stopped against a fret.</desc>
         <datatype>
-          <rng:ref name="data.FRETNUMBER"/>
+          <rng:ref name="data.FRETNUMBER" />
+        </datatype>
+      </attDef>
+      <attDef ident="tab.line" usage="opt">
+        <desc xml:lang="en">Used in German lute tablature in cases where vertical positioning deviates from the norm which can be specified by <att>tab.align</att>. 
+        Indicates the position of the note on one of the horizontal strands corresponding to the <att>lines</att> attribute on <gi>staffDef</gi>. 
+        (Note that in this case, the lines are conceptual rather than visible).
+        </desc>
+        <datatype>
+          <rng:ref name="data.CLEFLINE" />
         </datatype>
       </attDef>
       <attDef ident="tab.string" usage="opt">
-        <desc xml:lang="en">Records which string is to be played.</desc>
+        <desc xml:lang="en">This attribute is deprecated in favour of <att>tab.course</att> and will
+          be removed in a future version. Records which string is to be played.</desc>
+        <datatype>
+          <rng:ref name="data.STRINGNUMBER" />
+        </datatype>
+      </attDef>
+      <attDef ident="tab.course" usage="opt">
+        <desc xml:lang="en">Records which course is to be played.</desc>
         <datatype>
-          <rng:ref name="data.STRINGNUMBER"/>
+          <rng:ref name="data.COURSENUMBER" />
```

Comment:
Schematron says: "The `<rng:ref>` to "`data.COURSENUMBER`" is broken: There is no such thing in the specs". Not sure why, since it is declared in a macroSpec. Maybe connected to the given mode there?

Reply:
Any hints appreciated :-) 
@lpugin @kepper @bwbohl?

Reply:
As you added the missing `type="dt"` it should work now. @musicEnfanthen please try again.